<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWindow</class>
 <widget class="QMainWindow" name="MyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="rysujFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Wype≈Çnianie</string>
    </property>
    <widget class="QCheckBox" name="checkBox1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Warstwa 1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Warstwa 2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Warstwa 3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Warstwa 4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Warstwa 5</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>normal mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>darken mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>lighten mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>difference mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XOR mode</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>normal mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>darken mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>lighten mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>difference mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XOR mode</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>normal mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>darken mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>lighten mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>difference mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XOR mode</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>normal mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>darken mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>lighten mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>difference mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XOR mode</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>normal mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>darken mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>lighten mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>difference mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XOR mode</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
